internal/backend/crypto/gpg/cli/encrypt.go

```diff
@@ -25,25 +28,37 @@ func (g *GPG) Encrypt(ctx context.Context, plaintext []byte, recipients []string
 		args = append(args, "--trust-model=always")
 	}
 
+	if len(recipients) == 0 {
+		debug.Log("recipients list is empty!")
```

Comment:
shouldn't this warrant an early return?

Reply:
> shouldn't this warrant an early return?

I'm not sure. See my comment to your other comment.

Reply:
This should be an early return. If that makes tests fail we need to fix them.

Reply:
> This should be an early return. If that makes tests fail we need to fix them.

Done.